### DISCUSSION TITLE
gameobject_quest_item_binding

### DISCUSSION THREAD

**Discussion by cressidagp** [arcemu/db#29] [Q&A] [answered]
Hi there Community!

Im confused about this table... i saw many quest like: 'Millys Harvest', 'Galgars Cactus', 'Scavenging Deathknell' wich have and entry in gameobject_quest_pickup_binding also have an entry in gameobject_quest_item_binding. 

Its should be this way?

**Comment by dfighter1985** [comment, marked as answer]
> Hi there Community!
> 
> Im confused about this table... i saw many quest like: 'Millys Harvest', 'Galgars Cactus', 'Scavenging Deathknell' wich have and entry in gameobject_quest_pickup_binding also have an entry in gameobject_quest_item_binding.
> 
> Its should be this way?

Well, after digging the source code for a bit I've realized that no it shouldn't be that way.

gameobject_quest_item_binding is for gameobjects that contain quest items.
gameobject_quest_pickup_binding is for gameobjects that you need to interact with for a quest, but it has a very misleading name. It misled me too :(

**Reply by cressidagp** [reply]
Very well then, thanks for clearing me out 🙂
